src/AspNetCoreModuleV2/CommonLib/exceptions.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+inline VOID ReportException(std::exception& exception)
+{
```

Comment:
I think we should only report exception for debug build and check whether debug flag enabled as the cost could be very high for release scenario. Have you tested the behavior for the Remarks scenario in https://msdn.microsoft.com/en-us/library/windows/desktop/aa363362(v=vs.85).aspx

Reply:
The debug flag is undiscoverable and hard to set. I agree that we shouldn't call OutputDebugStringA for every request, but this is only in the failure path.

Reply:
Maybe make it on by default would be better.